### ISSUE TITLE
InstantUnify auto-converts metal blocks from Chisel despite being blacklisted

### ISSUE THREAD

**Issue by N1ch0l2s** [KidsDontPlay/Yunifier#26]
When making Chiseled metal blocks such as Iron or Gold, despite the mod being in InstantUnify's blacklist, it still autoconverts the metal blocks. 

To recreate:
Load in an instance with Thermal Foundation, Chisel, InstantUnify, and optionally UniDict
Chisel any applicable metal block listed in InstantUnify's config (Iron, Gold, Copper, etc.) 
The chiseled block will be automatically converted to its original variant despite Chisel being in the mod's blacklist.

**Comment by MoJoCreatior** [comment]
disable use unidict in config. set config prefered mods to use the same list from unidicts config.

the problem is that instant unify tries to use uniDicts settings, but ONLY uniDicts settings, so it completely ignores any blacklist/whitelist settings in the instantunify cfg

i just ran into the issue today, and after like 40 minutes of testing that was the solution.

tl;dr
change **B:useUnidict=_true_** >>> **B:useUnidict=_false_**
set **S:preferredMods** to the same list from unidict.cfg

**Comment by N1ch0l2s** [comment]
Changing B:useUniDict=false worked, but I couldn't find anything about S:preferredMods in unidict.cfg. Only thing listing various mods was "S:ownerOfEverything."
